### PR TITLE
Feature/password channel revisions

### PR DESCRIPTION
**Pull request by wmwash**

### CODE REVIEW

File: myuw/templates/handlebars/banner/profile.html [hunk 1]
```diff
@@ -37,24 +37,33 @@ <h3 class="sr-only">Official name and student ID number</h3>
 
             {{#with password}}
             <div class="profile-passwords row">
-                <h3 class="sr-only">Password Information</h3>
-                {{#if has_active_med_pw}}
-                <div class="col-md-6 uwmedicine-password">
-                    <h4>UW Medicine Account</h4>
-                    {{#if med_pw_expired }}
-                        <span class="password-alert">Password expired on {{formatDateAsDate expires_med}}. <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change your password to regain access.</a></span>
```
Comment (klpittman) [inline myuw/templates/handlebars/banner/profile.html]: @wmwash Not sure if you want this, but you could add `target="_blank"` to open these links in a new window.

File: myuw/templates/handlebars/banner/profile.html [hunk 2]
```diff
@@ -37,24 +37,33 @@ <h3 class="sr-only">Official name and student ID number</h3>
...
+                            <span class="password-alert">Password expired on {{formatDateAsDate expires_med}}. <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change your password to regain access.</a></span>
+                        {{ else }}
+                            <span class="password-status {{#if expires_in_30_days_or_less}}password-alert{{/if}}">Password expires in {{days_before_med_pw_expires}} days ({{formatDateAsDate expires_med}})</span><br>
+                            <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change UW Medicine password</a>
+                        {{/if}}
+                            <p>This password is for UW Medicine applications like Epic, ORCA, MINDscape, AMC network, etc.</p>
+                    </div>
+                    <div class="uwnetid-password col-md-6">
+                        <h4>UW NetID Account</h4>
+                        <span class="password-status">Password last updated {{days_after_last_pw_change}} days ago</span></br>
```
Comment (klpittman) [inline myuw/templates/handlebars/banner/profile.html]: You have a `</br>` at the end of this line. This should be either `<br>` or `<br />` if you want to be picky.

File: myuw/templates/handlebars/banner/profile.html [hunk 3]
```diff
@@ -37,24 +37,33 @@ <h3 class="sr-only">Official name and student ID number</h3>
...
+                        <a href="https://uwnetid.washington.edu/manage/?password">Change UW NetID password</a>
+                        <p>This password is for applications like UW email, MyUW, Web Publishing, etc.</p>
+                    </div>
+                    {{else}}
+                    <div class="uwnetid-password col-md-12">
+                        <h4>UW NetID Account</h4>
+                        <span class="password-status">Password last updated {{days_after_last_pw_change}} days ago</span></br>
```
Comment (klpittman) [inline myuw/templates/handlebars/banner/profile.html]: Another `</br>` here

File: myuw/templates/handlebars/banner/profile.html (hunk 1 above)
Comment (wmwash) [inline myuw/templates/handlebars/banner/profile.html]: Our current guideline for links in MyUW is that we only open things in a new window if the back button doesn't reliably get a user back to MyUW. That is, we assume any departure from MyUW is the workflow people are taking and MyUW usage is secondary. That may or may not be a good assumption for instructors; but maybe it is for staff?

Also, I've had on my mind to revisit this guideline based on a11y best practices.

**Pull request merged**